onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -751,6 +751,131 @@ def sample_inputs_unfold(op_info, device, dtype, requires_grad, **kwargs):
     yield opinfo_core.SampleInput(t, args=(dimension, size, step))
 
 
+def sample_inputs_slice_scatter(op_info, device, dtype, requires_grad, **kwargs):
+    del op_info
+    del kwargs
+    make_arg = functools.partial(
+        torch_testing.make_tensor, dtype=dtype, device=device, requires_grad=requires_grad
+    )
+
+    L = 20
+    cases = (
+        (
+            (L, L, L),
+            (
+                L,
+                L,
+                L,
```

Comment:
```suggestion
                L
```

nit: Remove the last comma to let black format the tuple into a compact line

Reply:
Below as well. Thanks!